wpilib/wpilib/shuffleboard/shuffleboard.py

```diff
@@ -0,0 +1,112 @@
+# validated: 2018-11-18 EN 175c6c1f0130 edu/wpi/first/wpilibj/shuffleboard/Shuffleboard.java
+
+from networktables import NetworkTablesInstance
+from .tab import ShuffleboardTab
+from .instance import ShuffleboardInstance
+
+
+__all__ = ['Shuffleboard']
+
+
+class Shuffleboard:
+    """
+    The Shuffleboard class provides a mechanism with which data can be added 
+    and laid out in the Shuffleboard dashboard application from a robot program.
+    Tabs and layouts can be specified, as well as choosing which widgets to 
+    display with and setting properties of these widgets; for example, 
+    programmers can specify a specific {@code boolean} value to be displayed 
+    with a toggle button instead of the default colored box, or set custom 
+    colors for that box.
+
+    For example, displaying a boolean entry with a toggle button:
+    .. code-block:: python
+        myBoolean = (Shuffleboard.getTab("Example Tab")
+            .add("My Boolean", False)
+            .withWidget("Toggle Button")
+            .getEntry())
+
+
+    Changing the colors of the boolean box:
+    .. code-block:: python
+        myBoolean = (Shuffleboard.getTab("Example Tab")
+            .add("My Boolean", False)
+            .withWidget("Boolean Box")
+            # TODO: WATWATWAT
```

Comment:
Haha. Please fix the example... should be `{'colorWhenTrue': 'green', .. }`

Reply:
I guess things like this are why we recommend that people that wish to help port changes actually know Java. :)